app/routes/events/route.tsx

```diff
@@ -22,7 +23,11 @@ export default function Page(): ReactNode {
 				<img alt="" className={styles.background} src="/images/background.webp" />
 			</picture>
 			<div className={styles.wrapper}>
-				<h1 className={styles.h1}>開催概要 / イベント詳細</h1>
+				<Header />
+				<h1 className={styles.h1}>
+					<Link to="/top#overview">開催概要</Link>
+					/ イベント詳細
```

Comment:
[must]

こうしないとスラッシュの左に空白が入らない

```suggestion
					{" "}/ イベント詳細
```